src/logic/safe/utils/spendingLimits.ts

```diff
@@ -168,7 +167,16 @@ export const addSpendingLimitBeneficiaryMultiSendTx = (beneficiary: string): Mul
     to: SPENDING_LIMIT_MODULE_ADDRESS,
     value: 0,
     data: spendingLimitContract.methods.addDelegate(beneficiary).encodeABI(),
-    operation: DELEGATE_CALL,
+  }
+}
+
+export const resetSpendingLimitBeneficiaryMultiSendTx = (beneficiary: string, token: string): MultiSendTx => {
```

Comment:
can we rename this function, the name indicates that this function will "do" the modification, but it's returning only the tx object to be sent.

Reply:
I was following the same naming as the other functions from the file. What would be a good name for it?

Reply:
Got it